### ISSUE TITLE
关于训练自己数据集的问题

### ISSUE THREAD

**Issue by VJoer** [PaddlePaddle/PaddleSeg#963]
我按照教程训练制作自己的数据集，使用Fast scnn 进行多分类语义分割训练，配置文件也是按照教程来的，但是训练结果只是二分类，请问是什么原因。
train_dataset:
  type: Dataset
  dataset_root: data
  train_path: data/train.txt
  num_classes: 25
  transforms:
    - type: ResizeStepScaling
      min_scale_factor: 0.5
      max_scale_factor: 2.0
      scale_step_size: 0.25
    - type: RandomPaddingCrop
      crop_size: [512, 512]
    - type: RandomHorizontalFlip
    - type: Normalize
  mode: train

**Comment by michaelowenliu** [comment]
@VJoer Hi, make sure the num_classes in the model is correct. https://github.com/PaddlePaddle/PaddleSeg/blob/release/v2.0/paddleseg/models/fast_scnn.py#L61

**Comment by VJoer** [comment]
> @VJoer Hi, make sure the num_classes in the model is correct. https://github.com/PaddlePaddle/PaddleSeg/blob/release/v2.0/paddleseg/models/fast_scnn.py#L61

好的，请问配置文件中如何修改传入图片的大小

**Comment by michaelowenliu** [comment]
@VJoer Hi, please take a look at our config files.

**Comment by MansGod** [comment]
您好，我也是用fastscnn训练，请问您预训练模型的参数怎么配置的呢，我修改了pretrained_model的路径，但是训练时报错无法在预训练模型中找到权重

**Comment by michaelowenliu** [comment]
@MansGod Hi, could you please put the error log?
